cmd/main.go

```diff
@@ -66,6 +66,8 @@ func main() {
 		&signal.Handler{}, // handle graceful termination
 	)
 
+	internal.LogFIPS() // Print the FIPS status
```

Comment:
At this point the intenal.Logger did not run the prerun phase, right?
How can this log be disabled then?
Should you run this as run.Prerunner as we do with the config-log?

Reply:
you're right. Fixed! :)